src/code/z_en_hy_code.c

```diff
@@ -35,11 +35,60 @@ static AnimationInfoS sAnimationInfo[] = {
     { &object_boj_Anim_005D9C, 1.0f, 0, -1, ANIMMODE_LOOP, -8 },
 };
 
-s8 gEnHyBodyParts[] = { -1, 1, 12, 13, 14, 9, 10, 11, 0, 6, 7, 8, 3, 4, 5, 2 };
+s8 gEnHyLimbToShadowBodyParts[ENHY_LIMB_MAX] = {
+    -1,                      //
+    ENHY_SHADOW_BODYPART_1,  //
+    ENHY_SHADOW_BODYPART_12, //
+    ENHY_SHADOW_BODYPART_13, //
+    ENHY_SHADOW_BODYPART_14, //
```

Comment:
Not sure what to comment for the limbs since there are multiple skeletons/limbs that need to align with this. Maybe just all of them with commas?